src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -584,8 +585,10 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
     return _.isEmpty(css) ? undefined : css;
   }
 
-  ngOnChanges() {
-    this.useData();
+  ngOnChanges(changes: SimpleChanges) {
+    if (changes.data && changes.data.currentValue) {
```

Comment:
Not required suggestion:
```suggestion
    if (changes.data?.currentValue) {
```

Reply:
I really like the syntactic sugar, but it makes backport not straightforward. :(

Reply:
I think we should keep the old style to make backporting easier.